src/web/app/components/question-submission-form/question-submission-form.component.scss

```diff
@@ -27,25 +27,13 @@
   margin-left: 20px;
 }
 
-.evaluee-col {
-  padding-left: 50px;
-  padding-right: 50px;
-}
-
-@media (max-width: 768px) {
-  .evaluee-col {
-    padding-left: 5px;
-    padding-right: 5px;
-  }
-}
-
 .evaluee-select {
   padding-left: 15px;
   padding-right: 15px;
 }
 
 .visibility-card {
-  background-color: rgba(0, 0, 0, .03);
+  background-color: rgba(0, 0, 0, 0.03);
```

Comment:
Turns out adding this causing linting to fail :P Do revert this.

Reply:
Ahh sorry, did not catch this! Will fix asap